<launch>
	<arg name="namespace" default="/merbots/whrov"/>
	<arg name="log2Console" default="false"/>
	<arg name="log2File" default=""/>

	<arg name="localAddr" default="1"/>
	<arg name="remoteAddr" default="2"/>
	
	<arg name="runComms" default="true"/>
	<arg name="linkType" default="halfd"/>

	<group if="$(arg runComms)">
		<node pkg="wireless_ardusub" name="op_v1" type="op_v1" respawn="true" ns="$(arg namespace)" output="screen">
			<param name="log2Console" value="$(arg log2Console)"/>
			<param name="localAddr" type="int" value="$(arg localAddr)"/>
			<param name="remoteAddr" type="int" value="$(arg remoteAddr)"/>
			<param name="linkType" type="str" value="$(arg linkType)"/>
			<param name="log2File" type="str" value="$(arg log2File)"/>
		</node>
	</group>
	<node pkg="whrov_hmi" name="whrov_hmi" type="whrov_hmi" respawn="false" ns="$(arg namespace)"/>
</launch>
